vimpager

```diff
@@ -539,6 +541,12 @@ main() {
     quit $?
 }
 
+expand_config_vars() {
+    for var in $config_vars; do
+        eval "$var=\"$(eval echo \"\$$var\")\""
+    done
```

Comment:
I know this is now extendable but at the moment it is three lines vs. two lines if we set the variables directly. And you had to introduce another artificial variable just to manage that. And you still have process substitution like this. I think it is easier to understand if you just do
```sh
eval runtime=\"$runtime\"
eval vimcat=\"$vimcat\"
```
```